server/sandbox_run_linux.go

```diff
@@ -830,14 +830,13 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 
 		container.SetMountPoint(mountPoint)
 
-		container.SetIDMappings(sandboxIDMappings)
-
 		container.SetSpec(g.Config)
 	} else {
 		log.Debugf(ctx, "dropping infra container for pod %s", sbox.ID())
 		container = oci.NewSpoofedContainer(sbox.ID(), containerName, labels, created, podContainer.RunDir)
 		g.AddAnnotation(ann.SpoofedContainer, "true")
 	}
+	container.SetIDMappings(sandboxIDMappings)
```

Comment:
can we add a comment here to future proof this?

Reply:
I kinda rely on the (pretty verbose) commit message I added that corresponds to this change (which can be found via git blame, or a series of git blames).

What would be the wording? Something like
```go
// needed for getSandboxIDMappings()
```
perhaps?

Reply:
fixed